src/pages/game/game.tsx

```
@@ -0,0 +1,33 @@
+import Pile from "../../shared/ui/Pile";
+import { randomCard } from "../../shared/utils/randomCard";
+import { Container, PileContainer } from "./style";
+import Dummy from "../../features/dummy/dummy";
+
+const Game = () => {
+  let deck = randomCard();
```

ESLint

[error] 'deck' is never reassigned. Use 'const' instead.

(prefer-const)